controllers/shipwrightbuild_controller.go

```diff
@@ -79,6 +81,19 @@ func (r *ShipwrightBuildReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		logger.Error(err, "Retrieving ShipwrightBuild object from cache")
 		return RequeueOnError(err)
 	}
+	init := b.Status.Conditions == nil
```

Comment:
it seems that the `init` variable is only used here and in the next line, better to just test in the `if` statement then.

Reply:
Need it to emphasis that we are in init state